content/docs/getting-started/technical-documentation/index.md

```diff
@@ -93,6 +94,13 @@ On the Overview page, click Docs in the tab bar. You should now see your documen
 
 ![a page of basic documentation for a service in the Backstage service catalog](./sample-service-docs-backstage.png)
 
+## Customize graphs in TechDocs
+
+When using mkdocs with TechDocs, base used for defaults such as plugins, theming etc are contained in [mkdocs-techdocs-core](https://github.com/backstage/mkdocs-techdocs-core)
+plugin. This means markdown-inline-graphviz extension is being used to replace Graphviz definitions with inline SVGs or PNGs. As long as `graphviz` is available in the environment
```

Comment:
This part about the environment is not useful for the reader. They have no control over the environment, we control that.